src/crimson/osd/osd.cc

```diff
@@ -1066,18 +1067,21 @@ seastar::future<> OSD::committed_osd_maps(version_t first,
       }
     });
   }).then([m, this] {
-    if (osdmap->is_up(whoami) &&
-        osdmap->get_addrs(whoami) == public_msgr->get_myaddrs() &&
-        bind_epoch < osdmap->get_up_from(whoami)) {
-      if (state.is_booting()) {
+    if (osdmap->is_up(whoami)) {
+      const auto up_from = osdmap->get_up_from(whoami);
+      logger().debug("osd.{} i'm up! up_from {}, bind_epoch {}, state {}",
+                     whoami, up_from, bind_epoch, state);
```

Comment:
```c++
logger().info("osd.{}: map e {} marked me up: up_from {}, bind_epoch {}, state {}",
              whoami, osdmap->get_epoch(), up_from, bind_epoch, state));
```

Reply:
@rzarzynski just a nit, but i don't feel strong either way. please feel free to merge this PR.

Reply:
Amended!